<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintConsole</class>
 <widget class="QDialog" name="PrintConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Printer Control</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>269</width>
     <height>293</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="xPlus10">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X+10</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="xPlus100">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X+100</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="yPlus100">
      <property name="text">
       <string>Y+100</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="yPlus10">
      <property name="text">
       <string>Y+10</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="zPlus10">
      <property name="text">
       <string>Z+10</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="yPlus1">
      <property name="text">
       <string>Y+1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="yPlus0_1">
      <property name="text">
       <string>Y+0.1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="homeY">
      <property name="text">
       <string>HomeY</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="yMinus0_1">
      <property name="text">
       <string>Y-0.1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="yMinus1">
      <property name="text">
       <string>Y-1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="yMinus10">
      <property name="text">
       <string>Y-10</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="zPlus1">
      <property name="text">
       <string>Z+1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="xPlus1">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X+1</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="xPlus0_1">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X+0.1</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="homeX">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>HomeX</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="xMinus0_1">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X-0.1</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="xMinus1">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X-1</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="xMinus10">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X-10</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="zPlus0_1">
      <property name="text">
       <string>Z+0.1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="homeZ">
      <property name="text">
       <string>HomeZ</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="zMinus0_1">
      <property name="text">
       <string>Z-0.1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="zMinus1">
      <property name="text">
       <string>Z-1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="zMinus10">
      <property name="text">
       <string>Z-10</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="yMinus100">
      <property name="text">
       <string>Y-100</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="xMinus100">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X-100</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="homeAll">
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
